### PR TITLE
LPS-102806 questions portlet 2

### PR DESCRIPTION
**Pull request by nhpatt**
Back again :D

### CODE REVIEW

File: modules/apps/questions/questions-web/package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+		"@clayui/sticker": "3.0.3",
+		"classnames": "2.2.6",
+		"frontend-js-react-web": "*",
+		"frontend-js-web": "*",
+		"prop-types": "15.7.2",
+		"react": "16.9.0",
+		"react-router-dom": "^5.0.1"
+	},
+	"description": "",
+	"main": "js/index.es.js",
```
Comment (boton) [inline modules/apps/questions/questions-web/package.json]: do you need main 🤔

File: modules/apps/questions/questions-web/package.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+		"@clayui/navigation-bar": "3.0.4",
+		"@clayui/sticker": "3.0.3",
+		"classnames": "2.2.6",
+		"frontend-js-react-web": "*",
+		"frontend-js-web": "*",
+		"prop-types": "15.7.2",
+		"react": "16.9.0",
+		"react-router-dom": "^5.0.1"
+	},
+	"description": "",
```
Comment (boton) [inline modules/apps/questions/questions-web/package.json]: remove description

File: modules/apps/questions/questions-web/src/main/resources/META-INF/resources/css/_question.scss [hunk 3]
```diff
@@ -0,0 +1,17 @@
+.question-headline {
+	margin-bottom: 0;
+}
+
+.question-answer {
+	border-top: 1px solid #E5E5E5;
+	padding: 0.5em;
```
Comment (boton) [inline modules/apps/questions/questions-web/src/main/resources/META-INF/resources/css/_question.scss]: Rem units :) https://j.eremy.net/confused-about-rem-and-em/

File: modules/apps/questions/questions-web/package.json (hunk 2 above)
Comment (nhpatt) [reply modules/apps/questions/questions-web/package.json]: https://github.com/boton/liferay-portal/pull/123/commits/b349df74b277312278fa5e9a9a43672cdd734fe7

Removed!

File: modules/apps/questions/questions-web/package.json (hunk 1 above)
Comment (nhpatt) [reply modules/apps/questions/questions-web/package.json]: No! :) https://github.com/boton/liferay-portal/pull/123/commits/b349df74b277312278fa5e9a9a43672cdd734fe7

File: modules/apps/questions/questions-web/src/main/resources/META-INF/resources/css/_question.scss (hunk 3 above)
Comment (nhpatt) [reply modules/apps/questions/questions-web/src/main/resources/META-INF/resources/css/_question.scss]: Got it! https://github.com/boton/liferay-portal/pull/123/commits/991a498c88fdbf9e652e29c65ca4cac84816de37 Thank you so much :)

**Pull request closed without merging**